pandas/tests/arrays/test_datetimelike.py

```diff
@@ -56,7 +56,68 @@ def timedelta_index(request):
     return pd.TimedeltaIndex(['1 Day', '3 Hours', 'NaT'])
 
 
-class TestDatetimeArray(object):
+class SharedTests(object):
+    index_cls = None
+
+    def test_take(self):
```

Comment:
We should not need to add such basic tests I think, as those are covered by the base Extension tests (we should of course test datetime specific aspects).

Is there anything in this test not tested by the base tests?

Reply:
The class-specific tests have tests for invalid fill_values